### DISCUSSION TITLE
Win32RenderingMode of Fluent Theme

### DISCUSSION THREAD

**Discussion by Estival-Dawn** [AvaloniaUI/Avalonia#15477] [Q&A] [answered]
In order to render VTK, I set
`.With(new Win32PlatformOptions { RenderingMode = new[] { Win32RenderingMode.Wgl } })`
but my Menu Item Flyout have weird white rectangular background. o(╥﹏╥)o
![{AF31FD57-1CE9-4878-B0ED-24F056DD0609}](https://github.com/AvaloniaUI/Avalonia/assets/160860179/faa614d8-66bd-4e9a-b1e5-4ce38077189f)
![Snipaste_2024-04-23_11-55-11](https://github.com/AvaloniaUI/Avalonia/assets/160860179/cd83139e-e971-4c1d-8c13-0868aa3e174a)

**Comment by maxkatz6** [comment, marked as answer]
Wgl doesn't support transparency and unlikely to do. 

In general, we do not recommend using Wgl at all.

**Comment by Estival-Dawn** [comment]
Thank you!
